CSharpOOP4/Shape.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace Kristina_Kulich__Application.CSharpOOP4;
+
+public abstract class Shape
+{
+ private string _name;
+
+ public Shape ()
+ { }
+
+ public Shape(string name)
+ {
+  _name = name;
+ }
+
+ public abstract double GetSquare();
+
+ public abstract void ShapeInfo();
```

Comment:
Bad name of the method. Method name should start from action. Something like GetShapeInformatin. 
Method should be virtual with implementation.

Reply:
Add logic to write to console name of the figure. And add this logic to override methods(base.ShapeInfo());

Reply:
fixed